File: src/api/Model/DTO/ChangePasswordDTO.cs
using Newtonsoft.Json;

namespace PhoneStoreManager.Model.DTO
{
    public class ChangePasswordDTO
    {
        [JsonProperty("old-pass")]
        public string OldPassword { get; set; } = string.Empty;

        [JsonProperty("new-pass")]
        public string NewPassword { get; set; } = string.Empty;

        [JsonProperty("re-new-pass")]
        public string ConfirmPassword { get; set;} = string.Empty;
    }
}

File: src/web/src/main/java/io/zoemeow/pbl6/phonestoremanager/model/dto/ChangePasswordDTO.java
package io.zoemeow.pbl6.phonestoremanager.model.dto;

import com.google.gson.annotations.SerializedName;

public class ChangePasswordDTO {
    @SerializedName("old-pass")
    private String oldPass;

    @SerializedName("new-pass")
    private String newPass;

    @SerializedName("re-new-pass")
    private String reNewPass;

    public String getOldPass() {
        return oldPass;
    }

    public void setOldPass(String oldPass) {
        this.oldPass = oldPass;
    }

    public String getNewPass() {
        return newPass;
    }

    public void setNewPass(String newPass) {
        this.newPass = newPass;
    }

    public String getReNewPass() {
        return reNewPass;
    }

    public void setReNewPass(String reNewPass) {
        this.reNewPass = reNewPass;
    }

    
}
